src/main/java/seedu/address/model/event/UniqueEventList.java

```diff
@@ -28,6 +28,14 @@ public class UniqueEventList implements Iterable<Event> {
     private final ObservableList<Event> internalUnmodifiableList =
             FXCollections.unmodifiableObservableList(internalList);
 
+    /**
+     * Returns the event in the event storage with the {@code eventName}.
+     */
+    public Event get(EventName eventName) {
```

Comment:
Maybe can be more specific on the name, i.e. getEvent

Reply:
i agree!